### PR TITLE
Update README and CONTRIBUTING docs

### PR DESCRIPTION
**Pull request by jessepeterson**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,65 +1,70 @@
...
-Your `GOPATH` must have thee subfolders - `bin`, `pkg` and `src`, and any code you create must live inside the `src` folder. It's also helpful to add `$GOPATH/bin` to your environment's `PATH` as that is where `go install` will place go binaries that you build.
+Go is a bit different from other languages in its requirements for how it expects its programmers to organize Go code files in directories.
+First, Go expects you to choose a folder, called a workspace (you can name it anything you'd like). The path to this folder must always be set in the environment variable `GOPATH` (example: `export GOPATH=/Users/groob/code/go`).
+Your `GOPATH` must have thee subfolders: `bin`, `pkg`, and `src`. Any code you create must live inside the `src` folder. It's also helpful to add `$GOPATH/bin` to your environment's `PATH` as that is where `go install` will place go binaries that you build. This makes it so that binaries that are insalled can just be invoked by name rathan their full page.
 
 Note: As of Go 1.8 the default `GOPATH` is set to `$HOME/go`.
```
Comment (groob) [inline CONTRIBUTING.md]: Let's move this note right after line 34 where we recommend setting a GOPATH

File: README.md [hunk 2]
```diff
@@ -1,59 +1,57 @@
...
-- MicroMDM aims to provide a declarative approach to device management. Too often vendor tools expect you to manage devices by filling out various forms in a web interface. While MicroMDM might have a web interface of it's own one day, the tool itself is inspired by popular DevOps processes like Configuration Management(chef/puppet/ansible,terraform) and orchestration frameworks(Kubernetes, Docker). 
-For example here is a process of applying a DEP profile: 
 ```
-$ micromdm apply dep-profile -f /tmp/profile.json
-updated profile CDACFE6013F926F5.
+$ mdmctl apply dep-profile -f /tmp/profile.json
+Defined DEP Profile with UUID 4B05B09E8AC7E7FC12C8F3338E099310
 
-$ micromdm get dep profile -uuid=CDACFE6013F926F5
+$ mdmctl get dep profile -f - -uuid=4B05B09E8AC7E7FC12C8F3338E099310
```
Comment (groob) [inline README.md]: It's `dep-profiles`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,65 +1,70 @@
...
-Your `GOPATH` must have thee subfolders - `bin`, `pkg` and `src`, and any code you create must live inside the `src` folder. It's also helpful to add `$GOPATH/bin` to your environment's `PATH` as that is where `go install` will place go binaries that you build.
+Go is a bit different from other languages in its requirements for how it expects its programmers to organize Go code files in directories.
+First, Go expects you to choose a folder, called a workspace (you can name it anything you'd like). The path to this folder must always be set in the environment variable `GOPATH` (example: `export GOPATH=/Users/groob/code/go`).
+Your `GOPATH` must have thee subfolders: `bin`, `pkg`, and `src`. Any code you create must live inside the `src` folder. It's also helpful to add `$GOPATH/bin` to your environment's `PATH` as that is where `go install` will place go binaries that you build. This makes it so that binaries that are insalled can just be invoked by name rathan their full page.
```
Comment (groob) [inline CONTRIBUTING.md]: `s/rathan/rather than`

File: README.md (hunk 2 above)
Comment (jessepeterson) [inline README.md]: Indeed! What's funny is two lines above this I changed it correctly. :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (jessepeterson) [inline CONTRIBUTING.md]: Er.. which specific note? The note about bin/pkg/src already comes after the GOPATH setting?

**Pull request merged**